src/common/BaseManager.ts

```diff
@@ -51,9 +57,11 @@ export default class BaseManager implements Manager {
         const parentEl = referenceEl.parentNode || documentEl;
 
         // Construct a layer element where we can inject a root React component
+        const isCurrent = getIsCurrentFileVersion(this.store.getState());
```

Comment:
I'm a little wary about injecting `store` into the managers, since it's already provided in `render`'s props. We'd also need to subscribe to it in case these values change, then clean up those subscriptions. I wonder if we should just allow a `resin` object option, instead? We already subscribe to the store in the annotators, right?